quickstart/www/assessment/standalone-assessment.php

```diff
@@ -0,0 +1,79 @@
+<?php
+    /**
+     * Copyright (c) 2017 Learnosity, MIT License
+     *
+     * Basic example of embedding a standalone assessment using Items API
+     * with `rendering_type: "assess"`.
+     */
+
+    // - - - - - - Section 1: server-side configuration - - - - - - //
+
+    // Include server side Learnosity SDK.
+    require_once __DIR__ . '/../../../src/vendor/autoload.php';
+    use LearnositySdk\Request\Init as LearnosityInit;
+    use LearnositySdk\Utils\Uuid as LearnosityUuid;
```

Comment:
Let's not rename those objects.
```suggestion
    use LearnositySdk\Request\Init;
    use LearnositySdk\Utils\Uuid;
```

Reply:
Updated. I couldn't use your commit for some reason, so I replicated it manually.